code/datums/diseases/advance/symptoms/viral.dm

```diff
@@ -119,3 +119,119 @@ Bonus
 	if(A.stealth >= 4) //more time before it's cured
 		power = 2
 	time_to_cure = max(A.resistance, A.stage_rate) * 10 * power
+
+/*
+//////////////////////////////////////
+
+Viral Suspended Animation
+
+	Very high stealth.
+	Abysmal resistance.
+	Poor stage speed.
+	Decent transmitability.
+	Medium Level.
+
+Bonus
+	The virus does not start until stage 5
+
+//////////////////////////////////////
+*/
+
+/datum/symptom/viralincubate
+	name = "Viral Suspended Animation"
+	desc = "The virus has very little effect until it reaches its final stage"
+	stealth = 4
+	resistance = -2
+	stage_speed = -2
+	transmission = 1
+	level = 4
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	prefixes = list("Asymptomatic ")
+	var/list/captives = list()
+	var/used = FALSE
+
+/datum/symptom/viralincubate/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.stage >= 5)
+		for(var/datum/symptom/S as() in captives)
+			S.stopped = FALSE
+			captives -= S
+		if(!LAZYLEN(captives))
+			stopped = TRUE
+	else if(!used)
+		for(var/datum/symptom/S as() in A.symptoms)
+			if(S.neutered)
+				continue
+			if(S == src)
+				continue
+			S.stopped = TRUE
+			captives += S
+		used = TRUE
+	
+
+/*
+//////////////////////////////////////
+
+Viral power multiplier
+
+	Average stats
+	UNOBTAINABLE- THIS SYMPTOM IS FOR ADMINBUS OR WEIRD CRAZY SHIT. You can enable it if you like, but be warned that this symptom is potentially very abusable
+
+Bonus
+	All unneutered symptoms in the virus have their power boosted
+
+//////////////////////////////////////
+*/
+/datum/symptom/viralpower
+	name = "Viral power multiplier"
+	desc = "The virus has more powerful symptoms. May have unpredictable effects"
+	stealth = 2
+	resistance = 2
+	stage_speed = 2
+	transmission = 2
+	level = -1 //currently unobtainable except by adminbus
+	prefixes = list("Super", "Mega", "Admin ")
+	var/maxpower
+	var/powerbudget
+	var/scramble = FALSE 
+	var/used = FALSE
+	threshold_desc = "<b>Transmission 8:</b> Constantly scrambles the power of all unneutered symptoms.<br>\
+					  <b>Stage Speed 8</b> Doubles the power boost"
+
+
+/datum/symptom/viralpower/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.speed >= 8)
+		power = 2
+	if(A.transmission >= 8)
+		scramble = TRUE
+
+/datum/symptom/viralpower/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	if(!used)
+		for(var/datum/symptom/S as() in A.symptoms)
+			if(S.neutered)
+				continue
+			S.power += power
```

Comment:
Couldn't this end up increasing its own power, causing symptoms after it in the list to get increased power as well?